cmd/cloud/server.go

```diff
@@ -513,16 +514,15 @@ func checkRequirements(logger logrus.FieldLogger) error {
 	}
 	logger.Infof("[startup-check] Using helm: %s", version)
 
-	// Check for extra tools that don't have a wrapper, but are still required.
-	extraTools := []string{
-		"kubectl",
+	kubectlClient, err := kubectl.New(silentLogger)
+	if err != nil {
+		return errors.Wrap(err, "failed kubectl client health check")
```

Comment:
nit: this could be more descriptive like `client does not exist` since it's really just doing a lookup that the binary exists in the PATH?

Reply:
I guess the error that you're wrapping states that... so maybe you don't even need to wrap it? Or just leave it.. 0/5